### PR TITLE
[mypyc] Build True, False and None op in irbuild

### PR DESCRIPTION
**Pull request by TH3CHARLie**
This PR builds three specialized op directly in irbuild, instead of using name_ref_ops.

### CODE REVIEW

File: mypyc/test-data/irbuild-set.test [hunk 1]
```diff
@@ -202,11 +202,12 @@ def update(s, x):
     s :: set
     x :: list
     r0 :: int32
-    r1, r2 :: None
+    r1 :: None
+    r2 :: None
```
Comment (TH3CHARLie) [inline mypyc/test-data/irbuild-set.test:206]: Although it's not something that important, I wonder how this happened.

Comment (JukkaL) [reply mypyc/test-data/irbuild-set.test:206]: This seems worth investigating. The underlying issue causing might be important.

File: mypyc/irbuild/ll_builder.py [hunk 2]
```diff
@@ -424,7 +425,24 @@ def call_union_item(value: Value) -> Value:
 
     def none(self) -> Value:
         """Load unboxed None value (type: none_rprimitive)."""
-        return self.add(PrimitiveOp([], none_op, line=-1))
+        res = self.alloc_temp(none_rprimitive)
+        value = self.add(LoadInt(1, -1, bool_rprimitive))
```
Comment (JukkaL) [inline mypyc/irbuild/ll_builder.py:429]: Use `none_rprimitive` as the type?

File: mypyc/primitives/misc_ops.py [hunk 3]
```diff
@@ -19,23 +19,23 @@
...
+# true_op = name_ref_op('builtins.True',
+#                       result_type=bool_rprimitive,
+#                       error_kind=ERR_NEVER,
+#                       emit=simple_emit('{dest} = 1;'))
+
+# # Get an unboxed False value
+# false_op = name_ref_op('builtins.False',
+#                        result_type=bool_rprimitive,
+#                        error_kind=ERR_NEVER,
+#                        emit=simple_emit('{dest} = 0;'))
```
Comment (JukkaL) [inline mypyc/primitives/misc_ops.py:38]: Remove commented-out code.

File: mypyc/test-data/analysis.test [hunk 4]
```diff
@@ -37,7 +37,7 @@ L4:
 L5:
     z = 2
 L6:
-    r5 = None
+    r5 = 1
     return r5
```
Comment (JukkaL) [inline mypyc/test-data/analysis.test:41]: Could this be simplified to `return 1`, similar to how we simplify constant value returns in other contexts?

**Pull request merged**